framework/include/mesh/FaceInfo.h

```diff
@@ -231,16 +238,25 @@ class FaceInfo
   const Point _neighbor_centroid;
   const Real _neighbor_volume;
 
-  /// Geometric weighting factor
-  const Real _gc;
-
   /// the distance vector between neighbor and element centroids
   const RealVectorValue _d_cf;
+
   /// the distance norm between neighbor and element centroids
   const Real _d_cf_mag;
+
   /// The unit normal vector pointing from element center C to element center F
   const RealVectorValue _e_cf;
 
+  /// Geometric weighting factor for face value interpolation
+  Real _gc;
```

Comment:
Yes, I agree. I'll change gc back to const for now. I accidentally left it like that since I was experimenting with having only one gc_ which changes values based on if the face connects skewed or non-skewed cells. In that case I compute a skewness indicator and assign gc_ based on that. The issue was that the current implementation uses gc = |neighbor_c - face-c|/( |elem_c-face_c| + |neighbor_c-face_c|), while I think we should switch to gc = |neighbor_c - face_intersect|/( |elem_c - neighbor_c|). The two are the same for non-skewed meshes but the last one is used for unstructured problems. I ended up keeping both because I believe that the transition would require regolding almost every test case. On long term, though if we want to switch the boundary handling from cell_center->face_center to cell_center->perpendicular_projection, this would be a natural addition.